odim-controller/scripts/odim-controller.py

```diff
@@ -1590,6 +1590,39 @@ def deploy_plugin(plugin_name):
 	# load existing hosts.yaml created for the deployment_id
 	load_k8s_host_conf()
 
+	# Validation for mandatory parameters in config file
+	pluginPackagePath = CONTROLLER_CONF_DATA['odimPluginPath'] + "/" + plugin_name
+	with open(pluginPackagePath + "/" + plugin_name + "-config.yaml", "r") as stream:
+			try:
+				pluginConf=yaml.safe_load(stream)
+				if pluginConf[plugin_name]['logPath'] == None or \
+					pluginConf[plugin_name]['username'] == None or \
+					pluginConf[plugin_name]['password'] == None:
+					logger.critical("mandatory parameter missing in Config file")
+					exit(1)
+				if plugin_name == "iloplugin":
```

Comment:
Donot use specific plugin name to compare, use suffix 'ServiceUUID' to fetch the details.